demos/dashy/tsconfig.json

```diff
@@ -19,6 +19,26 @@
         "types": []
     },
     "include": [
-        "src/**/*"
+        "./src/**/*"
+    ],
+    "references": [
+        {
+            "path": "../../packages/common/"
+        },
+        {
+            "path": "../../packages/comms/"
+        },
+        {
+            "path": "../../packages/graph/"
+        },
+        {
+            "path": "../../packages/marshaller/"
+        },
+        {
+            "path": "../../packages/other/"
+        },
+        {
+            "path": "../../packages/util/"
+        }
```

Comment:
Does dashy not need a reference to `../../packages/chart/`?

Reply:
Dashy has a ref to marshaller which in turn has a ref to chart